setup/prepare-crc.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#This script configures CRC for running app-studio in it
+#It cofigures the CRC to have better memory and CPUs 
+#It also configures CRC to have required addons.
+
+#TODO: Check for CRC installation and binary location and set it to CRC-BINARY
+CRCBINARY=$(readlink -f ~/.crc/bin/crc)
+
+echo $CRCBINARY
+#Get CRC VMs configured in local machine
+$CRCBINARY setup
+#Set memory and CPU allowance for CRC
+$CRCBINARY config set memory 16384
+$CRCBINARY config set cpus 6
+#enable netmetrices addons, to make sure member cluster has proper resources
+$CRCBINARY config set enable-cluster-monitoring true
+#Delete existing CRC cluster to apply the config updates
+$CRCBINARY delete
```

Comment:
Let's see if we can take a parameter for this script that will run $CRCBINARY delete -y if we're in a CI mode where we can't take interactive input.

Reply:
Ideally we should make the memory and CPU configurable via parameters.  However I would be OK with getting a hardcoded version in first and then adding that feature.

Reply:
The force-delete is moved to https://github.com/redhat-appstudio/infra-deployments/issues/214

Reply:
The configurable memory and CPU requirements is moved to https://github.com/redhat-appstudio/infra-deployments/issues/215